docs/pages/concepts/architectural/architecture.md

```diff
@@ -119,6 +120,10 @@ Meshery's Docker extension provides a simple and flexible way to design and oper
   <figcaption>Figure: Meshery Docker Extension</figcaption>
 </figure>
 
+## Meshery CLI 
+
+The command line interface ( also known as mesheryctl ) that is used to manage Meshery. Use mesheryctl to both manage the lifecycle of Meshery itself and to access and invoke any of Meshery's application and cloud native management functions.
```

Comment:
Please incorporate a link to either the mesheryctl reference or the using mesheryctl page.

Reply:
Can we also add a relevant image to it too or is it not required ?